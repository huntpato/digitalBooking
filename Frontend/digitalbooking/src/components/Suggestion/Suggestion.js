import styles from "./Suggestion.module.css";

export const Suggestion = (props) => {
  const { suggestion } = styles;
  return (
    <div className={suggestion}>
      <h5> Buenos Aires </h5>
      <h6>Argentina</h6>
    </div>
  );
};
