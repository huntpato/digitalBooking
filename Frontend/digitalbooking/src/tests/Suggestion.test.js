import React from 'react';
import { shallow } from 'enzyme';
import { Suggestion } from '../components/Suggestion/Suggestion'
import '@testing-library/jest-dom'
import { expect } from '@jest/globals';

describe('Pruebas en componente <Suggestion />', () => {
    const wrapper= shallow(<Suggestion/>);
   
    test('Comprobar que <Suggestion /> renderiza correctamente.', () => {
        expect( wrapper ).toMatchSnapshot();
    })

    test('Verificar que el componente carga la clase de CSS correctamente', () => {

        const div = wrapper.find('div');
        expect(div.hasClass('suggestion')).toBe(true);
    })
    
    
})
