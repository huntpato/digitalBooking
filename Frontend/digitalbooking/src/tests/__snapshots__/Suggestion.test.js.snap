// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en componente <Suggestion /> Comprobar que <Suggestion /> renderiza correctamente. 1`] = `
<div
  className="suggestion"
>
  <h5>
     Buenos Aires 
  </h5>
  <h6>
    Argentina
  </h6>
</div>
`;
